### DISCUSSION TITLE
Unit kvarh

### DISCUSSION THREAD

**Discussion by felix-baum-klingspor** [grafana/grafana#52495] [Dashboards, Panels & Visualizations]
There is an option to choose kWh and kvar as an unit but there is no option to choose kvarh. Would it be possible to add this unit, to complete the list?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
